src/main/java/dinkplugin/DinkPlugin.java

```diff
@@ -43,6 +52,15 @@
 )
 public class DinkPlugin extends Plugin {
 
+    private static final Color PINK = ColorUtil.fromHex("#eba2c0");
+    private static final Color RED = ColorUtil.fromHex("#f09d9e");
```

Comment:
Open to a different red (this was chosen from analogous on color wheel)

![image](https://user-images.githubusercontent.com/8106344/208213102-2c47db86-95eb-4624-9239-2a1ebed235c1.png)

Reply:
Not a readable color on default chat background
![image](https://user-images.githubusercontent.com/41973452/208741976-e2bd610f-bf0d-48ab-9fd5-7f9908e1d513.png)

Reply:
Good catch! will test alternatives later tonight

Reply:
Latest colors
![image](https://user-images.githubusercontent.com/8106344/209040215-885cf28b-bae7-4bc5-81a1-fea9fae1e0d9.png)